tests/test_2fa_duo.py

```diff
@@ -152,10 +156,10 @@ def test_2fa_session_single_crn(self, netid2, enter_duo_passcode):
             self.browser.get(self.shib_mfa_url)
             self.browser.send_inputs(netid2, self.password)
             self.browser.click(Locators.submit_button)
-            self.browser.wait_for_tag('p', "Using your 'sptest03' Duo identity.")
+            sleep(5)
```

Comment:
same ask as above (below?)

1. try using wait here if you haven't yet
2. add a comment indicating what's being waited for with the sleep